tests/test_opds.py

```diff
@@ -892,7 +949,7 @@ def test_active_loan_feed(self):
             x for x in links
             if x['rel'] == 'http://librarysimplified.org/terms/rel/user-profile'
         ]
-        annotator = cls(None, None, patron, test_mode=True)
+        annotator = cls(None, None, library=patron.library, patron=patron, test_mode=True)
```

Comment:
This is an unrelated bug in the test -- the third positional argument is the library, not the patron. Previously it didn't make a different but it does now because we always call _hidden_content_types on the library.